calc_app/views.py

```diff
@@ -143,22 +134,18 @@ class CategoryItemsView(View):
 
     def get_context(self, pk):
         """
-        Forms the context dictionary of variables to be rendered in a temolate
+        Forms the context dictionary of variables to be rendered in a template
         """
-        categories = Category.objects.order_by(
+        categories = models.Category.objects.order_by(
             "name"
         )  # all the categories to be inserted in the dropdown menu
-        items = ExpenseItem.objects.filter(
+        items = models.ExpenseItem.objects.filter(
             category_id=pk
         )  # items related to the current category
-        category = Category.objects.get(pk=pk)  # current category
-        category_months = (
-            {}
-        )  # month's that the items of the current category were bought
-        for item in items:
-            category_months[item.date.month] = get_month_name(item.date.month)
-        months = get_months_numbs_and_names()  # all the month's for the dropdown menu
-        form = ExpenseItemModelForm()  # form to create new instances of ExpenseItem
+        category = models.Category.objects.get(pk=pk)  # current category
```

Comment:
We have some misunderstanding
By `use absolute imports` I mean:
don't write `from .models import Model` 
do instead `from app.models import Model`